docs/2025/GSoC-projects.md

```diff
@@ -0,0 +1,547 @@
+---
+sidebar_position: 2
+title: Google Summer of Code Proposals 2025
+---
+
+<!--
+SPDX-License-Identifier: CC-BY-SA-4.0
+
+SPDX-FileCopyrightText: 2025 Kaushlendra Pratap <kaushlendra-pratap.singh@siemens.com>
+SPDX-FileCopyrightText: 2025 Siemens AG
+-->
+
+Welcome to the main page for all GSoC 2025 related information.
+
+- Check https://github.com/fossology/fossology/discussions/2908
+
+## Intro
+
+We from the fossology project would like to apply for GSoC 2025. Please see two
+main resources for finding out more FOSSology in general:
+
+- https://www.fossology.org
+- https://fossology.github.io/gsoc/
+
+Meetings: Checkout the [Meetings table](/docs/2025#meetings-table)
+
+## Interested in Application? - Getting Grip
+
+If you are interested in an application - great! We encourage your application.
+So the question is how to get started with the topic, just a few points:
+
+- Check [our guidelines for GSoC](/docs/guidelines/index.md)
+- Check https://www.fossology.org and these GitHub pages
+  https://github.com/fossology/fossology/wiki
+- Maybe check some [initial intro](https://github.com/fossology/fossology/wiki/New-at-FOSSology%2C-You-Could-...)
+    - https://youtu.be/TZqU5ZALI7U
+    - https://fossology.github.io/
+- Try to install fossology, either using vagrant or docker
+    - https://github.com/fossology/fossology/wiki/Install-from-Source
+- Read the proposed topics
+- Use the mailing list fossology-devel@fossology.org or contact proposed mentors
+  for further steps
+- [Slack invite link](https://join.slack.com/t/fossology/shared_invite/enQtNzI0OTEzMTk0MjYzLTYyZWQxNDc0N2JiZGU2YmI3YmI1NjE4NDVjOGYxMTVjNGY3Y2MzZmM1OGZmMWI5NTRjMzJlNjExZGU2N2I5NGY)
+- [GitHub discussion](https://github.com/fossology/fossology/discussions/2663)
+- If you are interested in trying to make contributions, see out issues with the
+  label [good first issue](https://github.com/fossology/fossology/contribute).
+  Maybe you could sort out the workflow and make a first pull request.
+
+## Examples from past programs
+
+In 2020, we were awarded seven slots, please see here what was the result of it:
```

Comment:
If possible, please update this section with 2024/2023 information in a different PR.

Reply:
We already have this information down below, Do you want me to remove that from here and add a new PR?